wp-modules/app/js/src/index.tsx

```diff
@@ -0,0 +1,4 @@
+import { render } from '@wordpress/element';
```

Comment:
We can use `render` from `@wordpress/element` instead of `react-dom`.

If we pin the required WP version to 6.2 again soon, we should consider updating [to use `createRoot`](https://github.com/reactwg/react-18/discussions/5). `createElement` [ is available](https://make.wordpress.org/core/2023/03/07/upgrading-to-react-18-and-common-pitfalls-of-concurrent-mode/) along with React 18 in 6.2.